service/policy/objects.proto

```diff
@@ -101,37 +111,45 @@ message Action {
 }
 
 /*
-   Subject Mapping (aka Access Control Subject Encoding aka ACSE):  Structures supporting the mapping of Subjects and Attributes (e.g. Entitlement)
+   Subject Mapping (aka Access Control Subject Encoding aka ACSE):  Structures
+   supporting the mapping of Subjects and Attributes (e.g. Entitlement)
 */
 
-// buflint ENUM_VALUE_PREFIX: to make sure that C++ scoping rules aren't violated when users add new enum values to an enum in a given package
+// buflint ENUM_VALUE_PREFIX: to make sure that C++ scoping rules aren't
+// violated when users add new enum values to an enum in a given package
 enum SubjectMappingOperatorEnum {
   SUBJECT_MAPPING_OPERATOR_ENUM_UNSPECIFIED = 0;
   // operator that returns true if a value in a list matches the string
   SUBJECT_MAPPING_OPERATOR_ENUM_IN = 1;
-  // operator that returns true if a value is not in a list that is matched by string
+  // operator that returns true if a value is not in a list that is matched by
+  // string
   SUBJECT_MAPPING_OPERATOR_ENUM_NOT_IN = 2;
   // operator that returns true if a value in a list contains the substring
   SUBJECT_MAPPING_OPERATOR_ENUM_IN_CONTAINS = 3;
 }
 
-// buflint ENUM_VALUE_PREFIX: to make sure that C++ scoping rules aren't violated when users add new enum values to an enum in a given package
+// buflint ENUM_VALUE_PREFIX: to make sure that C++ scoping rules aren't
+// violated when users add new enum values to an enum in a given package
 enum ConditionBooleanTypeEnum {
   CONDITION_BOOLEAN_TYPE_ENUM_UNSPECIFIED = 0;
   CONDITION_BOOLEAN_TYPE_ENUM_AND = 1;
   CONDITION_BOOLEAN_TYPE_ENUM_OR = 2;
 }
 
 /*
-   Subject Mapping: A Policy assigning Subject Set(s) to a permitted attribute value + action(s) combination
+   Subject Mapping: A Policy assigning Subject Set(s) to a permitted attribute
+   value + action(s) combination
 
-   Example: Subjects in sets 1 and 2 are entitled attribute value http://wwww.example.org/attr/example/value/one
-   with permitted actions TRANSMIT and DECRYPT
+   Example: Subjects in sets 1 and 2 are entitled attribute value
+   http://wwww.example.org/attr/example/value/one with permitted actions
+   TRANSMIT and DECRYPT
    {
     "id": "someid",
     "attribute_value": {example_one_attribute_value...},
-    "subject_condition_set": {"subject_sets":[{subject_set_1},{subject_set_2}]...},
-    "actions": [{"standard": "STANDARD_ACTION_DECRYPT"}", {"standard": "STANDARD_ACTION_TRANSMIT"}]
+    "subject_condition_set":
```

Comment:
I think we've been steadily removing examples from proto comments throughout, and it could be a good time to strip these examples off.